.travis.yml

```diff
@@ -5,6 +5,9 @@ notifications:
     on_success: change
     on_failure: always
 
+before_install:
+  - sed -i.bak -e 's|https://nexus.codehaus.org/snapshots/|https://oss.sonatype.org/content/repositories/codehaus-snapshots/|g' ~/.m2/settings.xml
```

Comment:
So you need snapshots for the build.
What is the snapshot dependency you are trying to get? Is it Apache Brooklyn?